### PR TITLE
Quick & dirty fix for a patient user

### PR DESCRIPTION
**Pull request by yvon-dblg**

### CODE REVIEW

File: packages/yourloops/components/patient-data.tsx [hunk 1]
```diff
@@ -60,35 +61,47 @@ function PatientDataPageError({ msg }: PatientDataPageErrorProps): JSX.Element {
...
 
   React.useEffect(() => {
-    if (blipApi === null) {
+    if (!initialized) {
       return;
     }
 
-    if (typeof patientId === "undefined") {
+    let patientId: string | null = null;
+    if (typeof paramPatientId === "string") {
```
Comment (leovalette) [inline packages/yourloops/components/patient-data.tsx:83]:
```suggestion
    if (paramPatientId) {
```
I guess you want to test that paramPatientId is not undefined, to me this way of writing is a bit confusing for the reader.

Comment (leovalette) [reply packages/yourloops/components/patient-data.tsx:83]: In fact, I think we can just write it like this:
`const  patientId: string | null = paramPatientId ?? userId`

Comment (yvon-dblg) [reply packages/yourloops/components/patient-data.tsx]: Resolved in my own way 😉

File: packages/yourloops/components/patient-data.tsx [hunk 2]
```diff
@@ -60,35 +61,48 @@ function PatientDataPageError({ msg }: PatientDataPageErrorProps): JSX.Element {
...
 
   React.useEffect(() => {
-    if (blipApi === null) {
+    log.debug("useEffect", { initialized, patient, userId, paramPatientId });
+    if (!initialized) {
       return;
     }
 
-    if (typeof patientId === "undefined") {
+    let patientId: string | null = null;
```
Comment (Nas84) [inline packages/yourloops/components/patient-data.tsx:83]: you could use the Nullish Coalescing (??) 
`let patientId: string | null = paramPatientId ?? userId;`

**Review by Nas84 — APPROVED** [review]
looks good to me. Apart a minor comment on patientId assignment.

**Pull request merged**